index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>SociLink | Login or Register</title>
+    <meta
+      name="description"
+      content="Welcome to SociLink! Your gateway to meaningful connections and engaging conversations. 
+      Log in or register today to join our growing community." />
+    <link
+      rel="stylesheet"
+      href="node_modules/bootstrap/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="/css/styles.css" />
+    <script
+      defer
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.bundle.min.js"></script>
+    <script defer src="/index.js"></script>
+  </head>
+  <body>
+    <div id="login">
+      <div class="space">
+        <div class="container-xxxl">
```

Comment:
What class is this `container-xxxl`?

Reply:
Oops, that's a typo. It's supposed to be "container-xxl" Fixed it. ~